src/box/session.h

```diff
@@ -375,6 +375,73 @@ generic_session_fd(struct session *session);
 int64_t
 generic_session_sync(struct session *session);
 
```

Comment:
Public functions must appear in `module.h`. For that you must export their declarations to this file, which is built automatically. Look for `/** \cond public */` and `/** \endcond public */` in the code. Did you test that patch in your own application? Because if `module.h` is not populated with the new functions, you simply couldn't see their declarations.